### PR TITLE
SISY-126: Replace reference to dlpsequencingdetail and included dry_run arg.

### PR DESCRIPTION
**Pull request by jeniferpham**

### CODE REVIEW

File: datamanagement/query_gsc_for_all_dlp_fastqs.py [hunk 1]
```diff
@@ -36,35 +37,39 @@ def main(storage_name, all, tag_name=None):
...
         # Re-get the sequencing details, may be unnecessary but safer
         # given that it is nested in sequencing and may have been changed
         # in a previous iteration of this loop
-        sequencingdetails = colossus_api.get('sequencingdetails', id=sequencing['dlpsequencingdetail']['id'])
+        sequencingdetails = colossus_api.get('sequencing', id=sequencing['id'])
 
         if sequencingdetails['gsc_library_id'] is not None:
             if sequencingdetails['gsc_library_id'] != import_info['gsc_library_id']:
-                raise Exception('gsc library id mismatch in dlpsequencingdetail {} '.format(sequencingdetails['id']))
+                raise Exception('gsc library id mismatch in sequencing {} '.format(sequencingdetails['id']))
```
Comment (amcpherson) [inline datamanagement/query_gsc_for_all_dlp_fastqs.py]: still referencing sequencingdetails, perhaps do a global search in sisypus to catch these

Comment (amcpherson) [reply datamanagement/query_gsc_for_all_dlp_fastqs.py]: oh, thats what the variable is called... rename the variable to just sequencing?

File: datamanagement/query_gsc_for_all_dlp_fastqs.py [hunk 2]
```diff
@@ -36,35 +37,39 @@ def main(storage_name, all, tag_name=None):
...
+            continue
 
         if import_info is None:
             continue
 
         # Re-get the sequencing details, may be unnecessary but safer
         # given that it is nested in sequencing and may have been changed
         # in a previous iteration of this loop
-        sequencingdetails = colossus_api.get('sequencingdetails', id=sequencing['dlpsequencingdetail']['id'])
+        sequencingdetails = colossus_api.get('sequencing', id=sequencing['id'])
```
Comment (amcpherson) [inline datamanagement/query_gsc_for_all_dlp_fastqs.py]: ```sequencing = colossus_api.get('sequencing', id=sequencing['id'])```

File: datamanagement/query_gsc_for_all_dlp_fastqs.py (hunk 1 above)
Comment (jeniferpham) [reply datamanagement/query_gsc_for_all_dlp_fastqs.py]: Hmm we're already using sequencing when we iterate through sequencing_list. Would sequencing_info be more appropriate?

**Pull request merged**